pkgs/by-name/sd/sdl3/package.nix

```
@@ -39,7 +39,7 @@
   ibusSupport ? stdenv.hostPlatform.isUnix && !stdenv.hostPlatform.isDarwin,
   jackSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
   libdecorSupport ? stdenv.hostPlatform.isLinux && !stdenv.hostPlatform.isAndroid,
-  openglSupport ? lib.meta.availableOn stdenv.hostPlatform libGL,
+  openglSupport ? !stdenv.hostPlatform.isWindows,
```

```suggestion
  openglSupport ? lib.meta.availableOn stdenv.hostPlatform libGL || stdenv.hostPlatform.isDarwin,
```

I think it'd be better to keep the current, slightly safer logic while adding a special case for Darwin

---

valid call, though i'd also argue this is just a default that can get an override. If it causes build issues on wonky platforms, it can get an overlay. And on those wonky platforms, hydra isn't building it anyways, so it doesn't even cost much to override here.

---

Also, why `!isWindows`? Windows has OpenGL, even if we currently don't.